h2o-py/requirements.txt

```diff
@@ -18,6 +18,9 @@ gssapi
 #pandas
 #scipy
 #sklearn
+#datatable
+#pyarrow
+#polars
```

Comment:
you can leave this here for info, but docker images use this file: https://github.com/h2oai/h2o-3/blob/master/h2o-py/test-requirements.txt
so they should be added there with a fix version, thanks

Reply:
Done

Reply:
> with a fixed version

same as other libraries.
You're reading too fast ;)